scripts/mirror.js

```diff
@@ -220,70 +218,59 @@ const bumpChromeAndroid = (originalData, sourceData, source) => {
 
 /**
  * @param {SupportStatement} originalData
- * @param {SupportStatement} sourceData
- * @param {string} source
+ * @param {SupportStatement} chromeData
+ * @param {SupportStatement} ieData
  * @returns {SupportStatement}
  */
-const bumpEdge = (originalData, sourceData, source) => {
-  let newData = {};
+const bumpEdge = (originalData, chromeData, ieData) => {
+  let newData = copyStatement(originalData);
```

Comment:
There's a bug here, or in `copyStatement`, which I don't think is really new, but it didn't manifest before. If you run `npm run mirror -- edge --modify=bool`, this change will be produced:

```diff
--- a/css/selectors/file-selector-button.json
+++ b/css/selectors/file-selector-button.json
@@ -26,11 +26,24 @@
             ],
             "edge": [
               {
+                "0": {
+                  "version_added": false
+                },
+                "1": {
+                  "version_added": "79",
+                  "alternative_name": "::-webkit-file-upload-button"
+                },
                 "version_added": false
               },
               {
-                "version_added": "79",
-                "alternative_name": "::-webkit-file-upload-button"
+                "0": {
+                  "version_added": false
+                },
+                "1": {
+                  "version_added": "79",
+                  "alternative_name": "::-webkit-file-upload-button"
+                },
+                "version_added": "79"
               }
             ],
             "firefox": {
```

It looks like Chrome's array of support statements is being treated as a single support statement, with broken results. It seems to me this might also be happening for other uses of `copyStatement`, but for this PR it's important to at least not regress what `npm run mirror` does.

Reply:
I don't know why it took me this long to finally get a fix out, especially one so easy.  Probably just needed a fresh pair of eyes...  Anyways, this is all fixed and ready to go!